packages/synapse-interface/utils/hooks/useBridgeListener.ts

```diff
@@ -54,5 +55,12 @@ export const useBridgeListener = () => {
     }
   }, [debouncedFromValue])
 
+  // Ensure destination address clears if settings turned off
+  useEffect(() => {
+    if (!showDestinationAddress && destinationAddress) {
+      dispatch(setShowDestinationAddress(null))
```

Comment:
false not null?

Reply:
thanks for catching this, fixed to dispatch the proper action in https://github.com/synapsecns/sanguine/pull/2137/commits/8e2c42698cd3993e7719dbe4e93e87e9e80fe198

we currently are dispatching the action from the Settings onClick toggle here: https://github.com/synapsecns/sanguine/blob/fe/destination-address/packages/synapse-interface/components/StateManagedBridge/SettingsSlideOver.tsx#L63

but I wanted to add an additional line in the bridge listener for a fallback check just incase Settings code gets updated in future (thinking here is that given the destination address feature is a sensitive feature for our Users, wrt to being extra careful about clearing dest. address store value)